### ISSUE TITLE
`@spartan-ng/cli:ui` does not work

### ISSUE THREAD

**Issue by hillin** [spartan-ng/spartan#518]
### Please provide the environment you discovered this bug in.

Angular 19.0.0
Angular CDK 19.0.2
@spartan-ng/ui-core: 0.0.1-alpha.370
@spartan-ng/cli: 0.0.1-alpha.370

### Which area/package is the issue in?

Don't know / other

### Description

```bash
❯ npx nx g @spartan-ng/cli:ui
Configuration file not found, creating a new one...
√ Choose a directory to place your spartan libraries, e.g. libs/ui · libs/ui
√ Choose which primitives you want to copy · all
In Nx 20, generating projects will no longer derive the name and root.
Please provide the exact project name and root in the future.
Example: nx g @nx/angular:library ui-ui-accordion-helm-ui-accordion-helm --directory libs/ui/ui-accordion-helm/ui-accordion-helm
# ...
Fetching prettier...

 NX   Cannot find module './libs/undefined/generator'

Require stack:
- node_modules\@spartan-ng\cli\src\generators\ui\generator.js
- node_modules\nx\src\config\schema-utils.js       
- node_modules\nx\src\command-line\run\executor-utils.js
- node_modules\nx\src\devkit-internals.js
- node_modules\nx\src\utils\assert-workspace-validity.js
- node_modules\nx\src\project-graph\build-project-graph.js
- node_modules\nx\src\project-graph\project-graph.js
- node_modules\nx\src\project-graph\file-utils.js  
- node_modules\nx\src\utils\package-manager.js     
- node_modules\nx\src\utils\package-json.js        
- node_modules\nx\bin\nx.js
```

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by goetzrobin** [comment]
This should be fixed by #516. It's already merged so I will test it and release a new CLI version!

**Comment by goetzrobin** [comment]
just released a fix for this in 0.0.1-alpha.373
